### DISCUSSION TITLE
Motivation to use better bridges

### DISCUSSION THREAD

**Discussion by drewtato** [voxeltycoon/issues#458] [General]
There is currently no reason besides aesthetic to use higher-tier bridges if you don't need the longer spans. In other games, the motivation to build better bridges is in speed: the more expensive bridges have a higher top speed for what's traveling over it.

I have two other ideas on how to incentivize higher-tier bridges.

1. Higher tier bridges should cost less maintenance. This would give a reason to replace old bridges, since new bridges would eventually pay for themselves.
2. If, in the future, we can use materials in addition to money to build things in the game, the low tier bridges can use resources that have no late-game uses. The low tiers can be made of brick, which would have no use after the early game. Then new tiers would be made of concrete, steel, or other more expensive materials. There would be no incentive later in the game to stay supplied with brick, since players would need the high-tier materials anyway when they need a longer bridge, and making lots of high-tier materials would be logistically simpler than making some high-tier and some low-tier materials.

**Comment by victorleblais** [comment]
Tunnels are very powerful compared to bridges, especially in the early-game where you can make tunnels as long as you want whilst you're stuck with the stone bridge with has poor width in between supports.

One way to make bridges better would be to make tunnels worse: make them more expensive, similar to how digging into the terrain gets expensive very fast. After all, you "have to dig" to make a tunnel so the price penalty should be closer to the digging one.

The length of the tunnel could have an impact on the price too (making one 200m long tunnel could be more expensive that two 100m tunnels).
